resources/elasticbeanstalk_configuration_options.go

```diff
@@ -0,0 +1,163 @@
+package resources
+
+import (
+	"context"
+	"time"
+
+	"github.com/aws/aws-sdk-go-v2/aws"
+	"github.com/aws/aws-sdk-go-v2/service/elasticbeanstalk"
+	"github.com/aws/aws-sdk-go-v2/service/elasticbeanstalk/types"
+	"github.com/cloudquery/cq-provider-aws/client"
+	"github.com/cloudquery/cq-provider-sdk/provider/schema"
+)
+
+func ElasticbeanstalkConfigurationOptions() *schema.Table {
+	return &schema.Table{
+		Name:         "aws_elasticbeanstalk_configuration_options",
+		Description:  "Describes the possible values for a configuration option.",
+		Resolver:     fetchElasticbeanstalkConfigurationOptions,
+		Multiplex:    client.AccountRegionMultiplex,
+		IgnoreError:  client.IgnoreAccessDeniedServiceDisabled,
+		DeleteFilter: client.DeleteAccountRegionFilter,
+		Options:      schema.TableCreationOptions{PrimaryKeys: []string{"application_arn", "date_created", "name", "namespace"}},
+		Columns: []schema.Column{
+			{
+				Name:        "account_id",
+				Description: "The AWS Account ID of the resource.",
+				Type:        schema.TypeString,
+				Resolver:    client.ResolveAWSAccount,
+			},
+			{
+				Name:        "region",
+				Description: "The AWS Region of the resource.",
+				Type:        schema.TypeString,
+				Resolver:    client.ResolveAWSRegion,
+			},
+			{
+				Name:        "application_arn",
+				Description: "The arn of the associated application.",
+				Type:        schema.TypeString,
+				Resolver:    schema.PathResolver("ApplicationArn"),
+			}, {
+				Name:        "date_created",
+				Description: "The date when the application was created.",
+				Type:        schema.TypeTimestamp,
+			},
+			{
+				Name: "change_severity",
+				Description: `An indication of which action is required if the value for this configuration option changes:  
+				* NoInterruption : There is no interruption to the environment or application availability.  
+				* RestartEnvironment : The environment is entirely restarted, all AWS resources are deleted and recreated, and the environment is unavailable during the process.  
+				* RestartApplicationServer : The environment is available the entire time`,
```

Comment:
I think we don't need this level of detail. First sentence is enough to convey the general idea.

Reply:
This was generated from cq-gen. Should I edit it?

Reply:
you can edit it yes